lib/experimental/blocks.php

```diff
@@ -183,3 +183,18 @@ function gutenberg_render_block_connections( $block_content, $block, $block_inst
 	}
 	add_filter( 'render_block', 'gutenberg_render_block_connections', 10, 3 );
 }
+
+
+function add_navigation_overlay_area( $areas ) {
+	$areas[] = array(
+		'area'        => 'navigation-overlay',
+		'label'       => _x( 'Navigation Overlay', 'template part area' ),
+		'description' => __(
+			'An area for navigation overlay content.'
+		),
+		'area_tag'    => 'section',
+		'icon'        => 'handle',
+	);
+	return $areas;
+}
+add_filter( 'default_wp_template_part_areas', 'add_navigation_overlay_area', 10, 1 );
```

Comment:
Hacking this in for now as a means to identify whether the template part is an "overlay".

We need this information in order that the Nav block can determine whether or not to show/hide various controls (e.g. Overlay controls!).

If we don't want a new area, we could use post meta to denote the "overlay" concept, but I am not sure if/how that's possible with Core Data currently.

Reply:
This is actually smart as the overlay can be a distinct area indeed.